packages/kit/src/exports/vite/index.js

```diff
@@ -298,6 +298,23 @@ async function kit({ svelte_config }) {
 						`${kit.files.routes}/**/+*.{svelte,js,ts}`,
 						`!${kit.files.routes}/**/+*server.*`
 					],
+					esbuildOptions: {
+						plugins: [
+							{
+								name: 'vite-plugin-sveltekit-setup:optimize',
+								setup(build) {
+									if (!kit.experimental.remoteFunctions) return;
+
+									const filter = new RegExp(
+										`.remote(${kit.moduleExtensions.join('|')})`.replaceAll('.', '\\.')
```

Comment:
To prevent this matching `asd.remote.ts/somethingelse.js` (highly unlikely but better safe than sorry)
```suggestion
										`.remote(${kit.moduleExtensions.join('|')})$`.replaceAll('.', '\\.')
```